transformer_engine/pytorch/csrc/userbuffers/userbuffers-host.cpp

```diff
@@ -7,19 +7,22 @@
 #include "ipcsocket.cc"
 #include "ipcsocket.h"
 #include "userbuffers.h"
+#include "gdrapi.h"
 #include <assert.h>
 #include <chrono>
 #include <cuda_runtime.h>
 #include <cuda_runtime_api.h>
+#if defined(__x86_64__)
```

Comment:
From what I can tell these arch macros are mainly supported on GCC and Clang. We don't support Windows, so breaking MSVC doesn't seem like a bit issue to me.